### PR TITLE
Update our gitignore

### PR DESCRIPTION
**Pull request by roberth1988**
Just a simple changes but adding for example .DS_Store is always a pain.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +15,20 @@
...
+
+# Test binary, build with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
+.glide/
+
```
Comment (nrwiersma) [inline .gitignore:27]: We use Go mod, this is outdated.

```suggestion
```

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,12 @@
+# Mac OS X files
+.DS_Store
+
 # IDE commons
 .idea
+.vscode/*
+.history/
```
Comment (nrwiersma) [inline .gitignore:7]: What is this for?

**Review by nrwiersma — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (roberth1988) [reply .gitignore:7]: Thats from vscode. Some of my colleagues are using that for go coding too.

File: .gitignore (hunk 1 above)
Comment (roberth1988) [reply .gitignore]: Ok removed that. Was from my default one ;)

**Review by nrwiersma — APPROVED** [review]
LGTM

**Pull request merged**